### PR TITLE
Adds `drawtol` to include a draw tolerance between solvers

### PR DESCRIPTION
**Pull request by lrsantos11**
PR according to  #52.

### CODE REVIEW

File: src/performance_profiles.jl [hunk 1]
```diff
@@ -37,16 +48,16 @@ function performance_ratios(T :: Array{Float64,2}; logscale :: Bool=true)
...
 """Produce the coordinates for a performance profile.
 
 There is normally no need to call this function directly. See the documentation
 of `performance_profile()` for more information.
 """
 function performance_profile_data(T :: Array{Float64,2}; logscale :: Bool=true,
-                                  sampletol :: Float64 = 0.0)
-    (ratios, max_ratio) = performance_ratios(T, logscale=logscale)
+                                  sampletol :: Float64 = 0.0, drawtol :: Float64 = 0.0)
+    (ratios, max_ratio) = performance_ratios(T, logscale=logscale,drawtol=drawtol)
```
Comment (abelsiqueira) [inline src/performance_profiles.jl:60]:
```suggestion
    (ratios, max_ratio) = performance_ratios(T, logscale=logscale, drawtol=drawtol)
```

File: src/performance_profiles.jl [hunk 2]
```diff
@@ -26,6 +26,17 @@ function performance_ratios(T :: Array{Float64,2}; logscale :: Bool=true)
   for p = 1 : np
     r[p, :] = T[p, :] / minperf[p];
   end
+  # Use a draw tolerance of `drawtol` (in percentage).
+  if (drawtol > 0. && drawtol < 1.)
```
Comment (abelsiqueira) [inline src/performance_profiles.jl:30]: Why the limit `< 1`?
Also
```suggestion
  if (drawtol > 0 && drawtol < 1)
```

File: src/performance_profiles.jl [hunk 3]
```diff
@@ -26,6 +26,17 @@ function performance_ratios(T :: Array{Float64,2}; logscale :: Bool=true)
...
+    drawtol += 1. 
+    for p in 1 : np
+      for s in 1 : ns
+        if r[p,s] <= drawtol
+          r[p,s] = 1.
+        end
+      end
+    end
```
Comment (abelsiqueira) [inline src/performance_profiles.jl:38]: I think this can be simplified to
```suggestion
    r[r .<= drawtol] .= 1
```

**Review by abelsiqueira — CHANGES_REQUESTED** [review]
Thanks for the PR. I've made a few comments. Also, can you upload an example of the difference of `drawtol = 0` and a small positive `drawtol`, like `0.01`?

File: src/performance_profiles.jl (hunk 3 above)
Comment (lrsantos11) [reply src/performance_profiles.jl:38]: Great!

**Comment by lrsantos11** [comment]
> Also, can you upload an example of the difference of `drawtol = 0` and a small positive `drawtol`, like `0.01`?

Here they are. I used `drawtol = 0.1`.

Solvers
Solvers_Draw10percent

**Review by dpo — APPROVED** [review]
Great addition! Thanks!

**Review by abelsiqueira — APPROVED** [review]
That's great, thanks.

**Pull request merged**